config/docker-template.xml

```diff
@@ -0,0 +1,146 @@
+                              <account-manager>
+                                <id>default-account-manager</id>
+                              </account-manager>
+                            </passkeys>
+                        </authenticator>
+                    </authenticators>
+                </authentication-service>
+            </settings>
+        </profile>
+        <profile>
+            <id>token-service</id>
+            <type xmlns:as="https://curity.se/ns/conf/profile/oauth">as:oauth-service</type>
+            <expose-detailed-error-messages/>
+            <settings>
+                <authorization-server xmlns="https://curity.se/ns/conf/profile/oauth">
+                    <client-store>
+                        <config-backed>
+                            <client>
+                                <id>haapi-ios-client</id>
+                                <client-name>Haapi iOS Client</client-name>
+                                <no-authentication>true</no-authentication>
+                                <redirect-uris>app:start</redirect-uris>
+                                <user-authentication>
+                                </user-authentication>
+                                <scope>address</scope>
+                                <scope>email</scope>
+                                <scope>openid</scope>
+                                <scope>phone</scope>
+                                <scope>profile</scope>
+                                <user-authentication>
+                                    <allowed-authenticators>HtmlForm</allowed-authenticators>
+                                    <allowed-authenticators>Passkeys</allowed-authenticators>
+                                </user-authentication>
+                                <capabilities>
+                                    <code/>
+                                    <haapi>
+                                        <issue-token-bound-authorization-code>true</issue-token-bound-authorization-code>
+                                    </haapi>
+                                </capabilities>
+                                <attestation>
+                                <disable-attestation-validation>true</disable-attestation-validation>
+                                <ios>
+                                    <app-id>$APPLE_BUNDLE_ID</app-id>
+                                    <ios-policy>ios-dev-policy</ios-policy>
+                                </ios>
+                                </attestation>
+                            </client>
+                            <client>
+                                <id>haapi-android-client</id>
+                                <client-name>Haapi Android Client</client-name>
+                                <no-authentication>true</no-authentication>
+                                <redirect-uris>app:start</redirect-uris>
```

Comment:
Is this the correct redirect URI? I get this issue when running the Android version: 
<img width="699" alt="image" src="https://github.com/user-attachments/assets/8d80f2d0-1bc1-4cdf-ba1d-687e7d20dfae" />

Reply:
Ah - you are right - I changed my mind on the value to use a couple of times since there was an issue applying the default value from the HAAPI module with RESTCONF. Fixed now.